File: test/hello_world.cpp
// Copyright 2011 The Emscripten Authors.  All rights reserved.
// Emscripten is available under two separate licenses, the MIT license and the
// University of Illinois/NCSA Open Source License.  Both these licenses can be
// found in the LICENSE file.

#include <stdio.h>

class Test {}; // This will fail in C mode

int main() {
  printf("hello, world!\n");
  return 0;
}

File: test/hello_world.js
/**
 * @license
 * Copyright 2013 The Emscripten Authors
 * SPDX-License-Identifier: MIT
 *
 * Simple test JS program used by `test/jsrun.py` to verify that a JS engine
 * is working.
 */

'use strict';

if (typeof print === 'undefined') {
  global.print = console.log
}

print('hello, world!');
